src/main/java/model/figure/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package model.figure;
+
+import model.Figure;
+
+public class Rectangle extends Figure {
+    private final int firstSide;
+    private final int secondSide;
+
+    public Rectangle(String color, int firstSide, int secondSide) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public int calculateArea() {
```

Comment:
The method 'calculateArea' should be named in a way that reflects it's an action performed by the object, such as 'getArea' to comply with Java naming conventions.